azkaban-db/src/main/sql/create.validated_dependencies.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE validated_dependencies (
+  file_name         VARCHAR(100),
+  file_sha1         VARCHAR(40),
```

Comment:
Is this really a `"VAR"CHAR`? Isn't it fixed length string?

Reply:
True, both are fixed length.